defaults/config.js

```diff
@@ -88,6 +87,17 @@ module.exports = {
 	displayNetwork: true,
 
 	//
+	// Lock network
+	//
+	// If set to true, users will not be able to modify host, port and tls
+	// settings and will be limited to the configured network.
+	//
+	// @type     boolean
+	// @default  false
+	//
+	lockNetwork: false,
```

Comment:
That would require some restructuring, plus have a dropdown on the client show allowed servers. I'm not keen on doing that myself.